src/storage/v3/shard.cpp

```diff
@@ -340,6 +340,7 @@ Shard::~Shard() {}
 Shard::Accessor::Accessor(Shard &shard, Transaction &transaction)
     : shard_(&shard), transaction_(&transaction), config_(shard_->config_.items) {}
 
+// TODO(jbajic) Remove with next PR
```

Comment:
With which next PR?

Reply:
The talk we had about fixing how `CreateNodeRequest` should be polished